spec/unit/application/lookup_spec.rb

```diff
@@ -78,7 +78,7 @@
   end
 
 
-  context 'when asked to explain' do
+  context 'when using a proper fixture' do
```

Comment:
Uh - this is an odd name for a context, all labels should be meaningful as names of examples, now it is named after a technical implementation detail. The old name was better. Has it changed so it is not just about explain. If so perhaps say "when given a valid configuration"